docs/domains/Domain.json

```diff
@@ -117,6 +117,11 @@
           "description": "Customization affecting Kubernetes Service generated for this WebLogic cluster.",
           "$ref": "#/definitions/KubernetesResource"
         },
+        "maxConcurrentShutdown": {
+          "description": "The maximum WebLogic Server instances that will shutdown in parallel for this cluster when it is being partially shutdown by lowering its replica count.A value of 0 means there is no limit. Defaults to `spec.maxClusterConcurrentShutdown` (which in turn defaults to 1).",
```

Comment:
@rosemarymarano, can you please also review this description? Thanks. A couple of proposed edits on this description... 1. "The maximum" -> "The maximum number of". 2. "shutdown" -> "shut down". As a verb, it's "shut down", while "shutdown" is a noun. 3. "count.A value" -> "count. A value" (add space) 4. " (which in turn defaults to 1)" -> ", which defaults to 1".